bip-update-process.md

```diff
@@ -0,0 +1,706 @@
+```
+BIP: ?
+Title: Updated BIP Process
+Author: Murch <murch@murch.one>
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-Updated-BIP-Process
+Status: Draft
+Type: Process
+Created: ?
+License: BSD-2-Clause
+Post-History: https://github.com/murchandamus/bips/pull/2
+              https://gnusha.org/pi/bitcoindev/59fa94cea6f70e02b1ce0da07ae230670730171c.camel@timruffing.de/#t
+Requires: 123
+Replaces: 2
+```
+
+## Abstract
+
+This _Bitcoin Improvement Proposal (BIP)_ provides information about the preparation of BIPs and policies relating to
+the publication of BIPs. It replaces [BIP 2](bip-0002.mediawiki) with a streamlined process, and may be amended to
+address the evolving needs of the BIP process.
+
+## Motivation
+
+BIP 2 is over eight years old and was written when different concerns were pressing to the Bitcoin[^capitalization]
+community. The BIP process as defined by BIP 2 aimed to facilitate the design and
+activation of protocol changes. In the past years, BIPs have more often described interoperational standards beyond the base
+protocol. The community has debated repeatedly about the role of the BIP Editors, and aspects of the process
+specified by BIP 2 that did not seem to achieve the intended goals. This BIP sunsets aspects of the BIP 2 process
```

Comment:
Suggest keeping only the last sentence of this paragraph.

Also, as mentioned in https://github.com/bitcoin/bips/pull/1712#discussion_r1889257750, we are still seeing quite a few new BIPs for protocol changes.